README.md

```
@@ -1,7 +1,14 @@
-#  Chords LSL Visualizer
+#  Chords LSL Visualizer - User Guide
```

markdownlint-cli2 (0.18.1)

Multiple spaces after hash on atx style heading

(MD019, no-multiple-space-atx)